### ISSUE TITLE
Transition animations

### ISSUE THREAD

**Issue by Bellavene** [ianyh/Amethyst#1263]
I tried to find anything about it here, but all hopeless. Only some old code from 2017.

Please, could someone give a direction to me, where I can get the code or what I should change to achieve animations like in the videos from this [#1257](https://github.com/ianyh/Amethyst/issues/1257) report?

**Comment by Bellavene** [comment]
So it is a bug. But would it not be great to have this possibility built-in? Especially for presentations.

**Comment by ianyh** [comment]
As far as I know there is no way for amethyst to support animated window transitions.

**Comment by Bellavene** [comment]
With all respect, the bug above with Chrome states otherwise itself.
Bettertouchtool, hammerspoon, Pheonix etc. can animate, so should Amethyst if you wanted to :)

**Comment by BigBrotherJu** [comment]
@ianyh Why are there window movement animations in the #1257 video? Is there any option to enable smoother window movement?

**Comment by Bellavene** [comment]
> @ianyh Why are there window movement animations in the #1257 video? Is there any option to enable smoother window movement?

Yabai have animations

**Comment by ianyh** [comment]
I believe you have to disable SIP to get the animations, but if someone else has a way to implement it without disabling SIP I am fine with getting a contribution.

**Comment by BigBrotherJu** [comment]
> Yabai have animations

@Bellavene I assume the poster of #1257 installed Yabai and Amethyst. So with Yabai installed, window movements in Amethyst can also have animations?

**Comment by Bellavene** [comment]
> I believe you have to disable SIP to get the animations, but if someone else has a way to implement it without disabling SIP I am fine with getting a contribution.
Agreed


> > Yabai have animations
> 
> @Bellavene I assume the poster of #1257 installed Yabai and Amethyst. So with Yabai installed, window movements in Amethyst can also have animations?

There is no reason to use both in the same time.

**Comment by ianyh** [comment]
> > Yabai have animations
> 
> 
> 
> @Bellavene I assume the poster of #1257 installed Yabai and Amethyst. So with Yabai installed, window movements in Amethyst can also have animations?

No, those animations were a result of an undocumented accessibility property being applied in some situations, and that collided with frame changes. I don't know how yabai does the animations, as I haven't looked.

**Comment by ianyh** [comment]
Closing this for now. If it is possible, it seems like a significant enough undertaking that I don't think I have the time to support it. I would consider a contribution, though.
